oauth-example/README.md

```diff
@@ -0,0 +1,58 @@
+# OAuth Example
+
+A minimal example showing how to hit an OAuth2 based API using an editor extension.
+The extension adds a menu item which hits Lucid's REST API to fetch and log the current user's Lucid folders.
+
+## Getting Started
+
+First install dependencies for the editor extension:
+```bash
+/oauth-example/editorextensions/oauth-example$ npm i
+```
+
+Next, create a new package in Lucid's developer portal, then create an OAuth 2.0 Client.
+
+![Create your package and OAuth client](https://cdn-cashy-static-assets.lucidchart.com/open-source-github-repositories/sample-lucid-extensions/oauth-client-setup.gif)
+
+Next, add a Redirect URI to your OAuth 2 client.
+The URI should be `https://extensibility.lucid.app/packages/PACKAGE_ID/oauthProviders/lucid/authorized` where `PACKAGE_ID` is replaced with your actual package id.
```

Comment:
nit: since we are creating a new package in the above step to create an oauth, it might be confusing as to what package ID we are referring to. Maybe specify it's the package ID of the original package that we want to redirect after OAuth authorization.

Reply:
We do call it out here:
https://lucid.readme.io/docs/editor-extension-using-oauth-apis#configure-lucids-redirect-url-in-oauth-provider